recipes/ppqsort/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy, replace_in_file
+from conan.tools.build import check_min_cppstd
+
+import os
+
+required_conan_version = ">=2.9.0"
+
+class PPQSortConan(ConanFile):
+    name = "ppqsort"
+    package_type = "header-library"
+
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/GabTux/PPQSort"
+    description = "Parallel Pattern Quicksort"
+    topics = ("algorithms", "sorting", "parallel", "header-only")
+
+    implements = ["auto_header_only"]
+    settings = "os", "arch", "compiler"
```

Comment:
```suggestion
    no_copy_sources = True
```

Can be safely removed, and the new attribute will avoid a duplication of folders

Reply:
Should I keep `compiler` to check minimum standard version?